### ISSUE TITLE
Missing genKey.js for npm install

### ISSUE THREAD

**Issue by dbauszus-glx** [mcollina/fastify-secure-session#8]
Hi Matteo,

I am struggling to get this working.

`npm i fastify-secure-session`  errs with following message on top of the stack.

npm ERR! path /home/dennis/Git/xyz/node_modules/fastify-secure-session/genKey.js

Is this a dependency which I need to install manually?

**Comment by mcollina** [comment]
which version of npm are you using? I cannot reproduce this, and `npm i fastify-secure-session` completes correctly here.

**Comment by dbauszus-glx** [comment]
I am using npm v6.0. The complete error message is as follows.

```
dennis@xps:~/Git/xyz$ npm i fastify-secure-session
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! path /home/dennis/Git/xyz/node_modules/fastify-secure-session/genKey.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/home/dennis/Git/xyz/node_modules/fastify-secure-session/genKey.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/dennis/.npm/_logs/2018-06-12T09_40_29_819Z-debug.log
```

**Comment by mcollina** [comment]
Fixed in 0.4.2, thanks!
